### PR TITLE
Fix Issue #515

### PR DESCRIPTION
**Pull request by kaie**
Fall back to date only parsing

### CODE REVIEW

**Comment by kaie** [comment]
@darktrojan could you please approve the workflows and have a look? Thanks

**Comment by darktrojan** [comment]
I don't have those powers. @kewisch?

File: lib/ical/time.js [hunk 1]
```diff
@@ -1150,11 +1150,24 @@
...
+
+    try {
+      if (aValue.length > 10) {
+        result = ICAL.Time.fromDateTimeString(aValue, aProperty);
+        // If we arrive here, parsing worked, we can skip the fallback.
+        useDateOnly = false;
+      }
+    } catch (e) {}
+
+    if (useDateOnly) {
```
Comment (kewisch) [inline lib/ical/time.js:1164]: Let's gate the leniency on `!ICAL.design.strict`

**Review by kewisch — CHANGES_REQUESTED** [review]
A test would be marvelous. 

I'm not sure why the linters are failing, but at least on this branch we use var (almost) everywhere for backwards compat. I need to merge the ES6 branch before it diverts too much, here we can use all the modern things.

I suspect we passed the deadline you were looking to make? Apologies in advance. If not then I'd be fine skipping the test in order to push this out, as long as you can follow up with the test a bit later.

**Comment by kaie** [comment]
abandoning per request from Lasana

**Pull request closed without merging**